File: MapAccounts/MapAccounts/Client/Scripts/Primitives/PointDTO.js
function PointDTO()
{
    this.ID = -1;
    this.lat = -1;
    this.lng = -1;
}

PointDTO.initialize = function(_ID, _lat, _lng)
{
    var ret = new PointDTO();
    ret.ID = _ID;
    ret.lat = _lat;
    ret.lng = _lng;
    return ret;
}

File: MapAccounts/MapAccounts/Models/Primitives/PointDTO.cs
namespace MapAccounts.Models.Primitives
{
    public class PointDTO
    {
        public PointDTO() { }

        public PointDTO(StreetPointModel point)
        {
            ID = point.ID;
            lat = point.lat;
            lng = point.lng;
        }

        public string ID { get; set; }
        public double lat { get; set; }
        public double lng { get; set; }
        public PanoramaDTO PanoramaDTO { get; set; }
    }
}